src/tools/tidy/src/extdeps.rs

```diff
@@ -27,7 +27,7 @@ pub fn check(root: &Path, bad: &mut bool) {
 
         // Ensure source is allowed.
         if !ALLOWED_SOURCES.contains(&&*source) {
-            tidy_error!(bad, "invalid source: {}", source);
+            //tidy_error!(bad, "invalid source: {}", source);
```

Comment:
This was only needed so I can test locally with the git deps; I'll remove it when we have a real usable chalk version upstream.

Reply:
did you to remove this before it lands?

Reply:
Ugh whoops. Forgot about this. Will do that in a bit.`